site/src/pages/HealthPage/DismissWarningButton.tsx

```diff
@@ -0,0 +1,72 @@
+import NotificationOutlined from "@mui/icons-material/NotificationsOutlined";
+import NotificationsOffOutlined from "@mui/icons-material/NotificationsOffOutlined";
+import { healthSettings, updateHealthSettings } from "api/queries/debug";
+import { useMutation, useQuery, useQueryClient } from "react-query";
+import { displaySuccess } from "components/GlobalSnackbar/utils";
+import LoadingButton from "@mui/lab/LoadingButton";
+import Skeleton from "@mui/material/Skeleton";
+import { HealthSection } from "api/typesGenerated";
+
+export const DismissWarningButton = (props: { healthcheck: HealthSection }) => {
+  const queryClient = useQueryClient();
+  const healthSettingsQuery = useQuery(healthSettings());
+  // They call the same mutation but are used in diff contexts so we don't want
+  // to merge their states
```

Comment:
Updated the comments to include a use case:
```tsx
They call the same mutation but are used in diff contexts so we don't want
to merge their states. Eg. You dismiss a warning and when it is done it
will show the enable button but since the mutation is still invalidating
other queries it will be in the loading state when it should be idle.
```